cms/test_utils/project/app_with_bad_cms_file/cms_config.py

```diff
@@ -0,0 +1 @@
+raise KeyError
```

Comment:
Is there a reason for the KeyError? maybe use a `RuntimeError` instead.
Also some comments here would be good.